key-vault-node-quickstart-v4/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "key-vault-node-quickstart",
+    "description": "Simple Hello World Node.js sample for Azure Key Vault using Managed Service Identity",
+    "version": "0.0.1",
+    "private": true,
+    "license": "MIT",
+    "author": "Microsoft",
+    "engines": {
+        "node": ">=6.9.1"
+    },
+    "scripts": {
+        "start": "node index.js"
+    },
+    "dependencies": {
+        "@azure/keyvault-secrets": "4.0.0-preview.4",
```

Comment:
When the release goes out, these versions will need to be updated

Reply:
Please update to GA version now that they are published.